accounts/abi/bind/backends/blockchain_test.go

```diff
@@ -281,7 +282,7 @@ func TestBlockChainSendTransaction(t *testing.T) {
 	state, err := bc.State()
 	txPoolConfig := blockchain.DefaultTxPoolConfig
 	txPoolConfig.Journal = "/dev/null" // disable journaling to file
-	txPool := blockchain.NewTxPool(txPoolConfig, bc.Config(), bc, &dummyGovModule{chainConfig: bc.Config()})
+	txPool := blockchain.NewTxPool(txPoolConfig, bc.Config(), bc, &dummyGovModule{chainConfig: bc.Config()}, []kaiax.TxPoolModule{})
```

Comment:
nit: `nil` is more concise.
```suggestion
	txPool := blockchain.NewTxPool(txPoolConfig, bc.Config(), bc, &dummyGovModule{chainConfig: bc.Config()}, nil)
```

Reply:
fix at https://github.com/kaiachain/kaia/pull/263/commits/0f6f9d4519d3e5bc0b5f0906f3eae01324bda366